server/services/communityApiService.ts

```diff
@@ -14,16 +14,42 @@ export interface DeliusUser {
 }
 
 export interface DeliusServiceUser {
-  offenderId: string
+  otherIds: OtherIds
+  offenderProfile: OffenderProfile
```

Comment:
I guess this was true before, but as this data structure gets larger we're making more and more assumptions about the data received in the response. At some point we'll need to validate these assumptions instead of just doing a type assertion, else we might end up passing malformed data to the interventions service. This can be covered within the scope of [IC-620](https://dsdmoj.atlassian.net/browse/IC-620); can we put a TODO for that?

Reply:
I've begrudgingly written my first TODO (you know how I feel about comments in code 🤐)

Reply:
Ugh, I've just tried testing this out and everything needs to be nullable here...